.dependency-cruiser.js

```diff
@@ -183,7 +238,140 @@ module.exports = {
           'npm-peer'
         ]
       }
-    }
+    },
+
+    /* application-specific architecture rules */
+    {
+      name: 'dependencies-of-shared',
+      severity: 'error',
+      from: {
+        path: dir('src/shared')
+      },
+      to: {
+        pathNot: [
+          dir('src/shared'),
+          dir('src/fs'),
+          ...npmDeps(
+            'node-fetch',
+            'markdown-escape',
+            'bn.js',
+          )
+        ]
+      }
+    },
+    {
+      name: 'dependencies-of-fs',
+      severity: 'error',
+      from: {
+        path: dir('src/fs')
+      },
+      to: {
+        pathNot: [
+          // TODO(SRe): is this okay?
+          dir('src/shared'), // 'shared' and 'fs' form a cycle
```

Comment:
See above. This direction is OK but the other isn't. But since it's only in test code, we may just need to be more specific.

Reply:
I've removed the comments.